snapshot_manager/snapshot_manager/config.py

```diff
@@ -135,9 +138,12 @@ def to_github_dict(self) -> dict:
         ...   copr_project_tpl="SomeProjectTemplate-YYYYMMDD",
         ...   copr_monitor_tpl="https://copr.fedorainfracloud.org/coprs/g/mycoprgroup/SomeProjectTemplate-YYYYMMDD/monitor/",
         ...   chroot_pattern="^(fedora-(rawhide|[0-9]+)|rhel-[8,9]-)",
-        ...   chroots=["fedora-rawhide-x86_64", "rhel-9-ppc64le"]
+        ...   chroots=["fedora-rawhide-x86_64", "rhel-9-ppc64le"],
+        ...   additional_copr_buildtime_repos=["copr://@fedora-llvm-team/llvm-test-suite", "https://example.com"]
```

Comment:
Why do we need `https://example.com` ?

Reply:
This is not the real repo to add. This is a doctest that proofs that we indeed get two entries if required.